csharp/class.go

```diff
@@ -11,6 +11,7 @@ type ClassDeclaration struct {
 	modifiers  []string
 	attributes []Writable
 	members    []Writable
+	summary    *SummaryDeclaration
```

Comment:
In general go tries to leverage `zero`/default values, rather than use null pointers. Generally it allows you to simplify the code quite a bit.